model_analyzer/model_manager.py

```diff
@@ -188,9 +195,26 @@ def _check_for_ensemble_model_incompatibility(
     def _init_state(self):
         """
         Sets ModelManager object managed
-        state variables in AnalyerState
+        state variables in AnalyzerState
         """
 
         self._state_manager.set_state_variable(
             'ModelManager.model_variant_name_manager',
             self._state_manager.default_encode(ModelVariantNameManager()))
+
+    def _check_for_valid_measurement(
+            self, measurement: Optional[RunConfigMeasurement]) -> None:
+        if measurement:
+            self._received_measurement_values_from_pa = True
+        else:
+            self._measurement_attempts += 1
+
+    def _stop_ma_if_no_valid_measurement_threshold_reached(self) -> None:
+        if self._received_measurement_values_from_pa:
+            return
+
+        if self._measurement_attempts >= INVALID_MEASUREMENT_THRESHOLD:
+            raise TritonModelAnalyzerException(
+                            f'The first {INVALID_MEASUREMENT_THRESHOLD} attempts to acquire measurements have failed. ' \
```

Comment:
Did this auto-format like this? Why is it so indented? It seems to be too wide, even for my ultra wide screen monitor.

Reply:
It did. I've fixed the formatting by breaking it into 3 lines. Auto-formatting is "funny" with fstrings, if I don't manually breakup the line it will let me continue way past the character limit.